### PR TITLE
Fix for Dissolve functionality

### PR DESCRIPTION
**Pull request by saravankumarpa**
*Issue #:*
https://github.com/bcgov/entity/issues/2676
*Description of changes:*

API for deleting the org is added
Logic in front end added
Event bus added so that notifications can be added

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/src/auth_api/exceptions/errors.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class Error(Enum):
     OWNER_CANNOT_BE_REMOVED = 'Owner cannot be removed by anyone', http_status.HTTP_400_BAD_REQUEST
     MAX_NUMBER_OF_ORGS_LIMIT = 'Maximum number of organisations reached', http_status.HTTP_400_BAD_REQUEST
     ALREADY_CLAIMED_PASSCODE = 'Passcode you entered has already been claimed', http_status.HTTP_406_NOT_ACCEPTABLE
+    ORG_CANNOT_BE_DISSOLVED = 'Organsiation cannot be dissolved', http_status.HTTP_406_NOT_ACCEPTABLE
```
Comment (sutherlanda) [inline auth-api/src/auth_api/exceptions/errors.py]: Spelling error here: organization

File: auth-web/src/App.vue [hunk 2]
```diff
@@ -26,6 +26,8 @@ import { mapActions, mapMutations, mapState } from 'vuex'
 import { AccountSettings } from '@/models/account-settings'
 import BusinessModule from '@/store/modules/business'
 import ConfigHelper from '@/util/config-helper'
+import { Event } from '@/models/event'
+import { EventBus } from '../src/event-bus'
```
Comment (sutherlanda) [inline auth-web/src/App.vue]: Can we switch to use the '@' import style to be consistent?

File: auth-web/src/components/auth/UserManagement.vue [hunk 3]
```diff
@@ -135,12 +136,16 @@ import { Component, Prop, Vue } from 'vue-property-decorator'
 import MemberDataTable, { ChangeRolePayload } from '@/components/auth/MemberDataTable.vue'
 import { mapActions, mapGetters, mapState } from 'vuex'
 import { Business } from '@/models/business'
+import ConfigHelper from '@/util/config-helper'
+import { Event } from '@/models/event'
+import { EventBus } from '../../event-bus'
```
Comment (sutherlanda) [inline auth-web/src/components/auth/UserManagement.vue]: Same comment as above ('@')

File: auth-web/src/locales/en.json [hunk 4]
```diff
@@ -52,5 +52,7 @@
   "profileDeactivatedTitle": "Your profile has been deactivated.",
   "profileDeactivatedMsg": "Your contact information, account associations and business affiliations have been removed.",
   "leaveTeamTitle": "You have been removed from this account.",
-  "leaveTeamMsg": "If you want to regain access to this account, you will need a new invitation from the account owner."
+  "leaveTeamMsg": "If you want to regain access to this account, you will need a new invitation from the account owner.",
+  "confirmLDissolveTeamTitle": "Confirm Dissolve account",
```
Comment (sutherlanda) [inline auth-web/src/locales/en.json]: Can we fix the typo in the key? Also, we should use consistent capitalization: "Confirm Dissolve Account"

File: auth-api/src/auth_api/services/org.py [hunk 5]
```diff
@@ -86,11 +86,29 @@ def update_org(self, org_info):
...
+        # Check authorization for the user
+        current_app.logger.debug('<org Inactivated')
+        check_auth(token_info, one_of_roles=OWNER, org_id=org_id)
+
+        org: OrgModel = OrgModel.find_by_org_id(org_id)
+        if not org:
+            raise BusinessException(Error.DATA_NOT_FOUND, None)
+
+        count_members = len([member for member in org.members if member.status in VALID_STATUSES])
+        if count_members > 1:
```
Comment (sutherlanda) [inline auth-api/src/auth_api/services/org.py]: Can we combine these two conditionals?

**Comment by saravankumarpa** [comment]
> Nice, this was a lot of work! A couple of very minor corrections but good to merge once those are done.

Thanks...Good feedbacks..appreciate it..Done the changes

**Pull request merged**